pkg/identityserver/bunstore/gateway_antenna_store.go

```diff
@@ -37,17 +37,13 @@ type GatewayAntenna struct {
 
 	Location
 
-	// TODO: Add attributes as JSONB field (https://github.com/TheThingsNetwork/lorawan-stack/issues/5614).
```

Comment:
Probably we should remove them from the `GatewayAntenna` model of `gormstore` too.

Reply:
The `gormstore` `GatewayAntenna` model has no mention of the `attributes`. My assumption is that it was added with the switch to `bunstore` as a preparation step.

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/blob/22db57b2fe35ee8f93281b9f6a9a737f97954288/pkg/identityserver/gormstore/gateway_antenna.go#L19-L27

Reply:
I see, in this PR I had already removed them; that's why I couldn't find it in my local version